src/LogicApp/LogicApp/Cmdlets/IntegrationAccount/UpdateAzureIntegrationAccountMapCommand.cs

```diff
@@ -135,16 +112,13 @@ public override void ExecuteCmdlet()
                 CmdletHelper.GetContentFromFile(this.TryResolvePath(this.MapFilePath));
             }
 
-            if (!string.IsNullOrEmpty(this.ContentType))
-            {
-                integrationAccountMapCopy.ContentType = this.contentType;
-            }
-
             if (!string.IsNullOrEmpty(this.MapType))
             {
                 integrationAccountMapCopy.MapType = this.MapType;
             }
 
+            integrationAccountMapCopy.ContentType = this.MapType.Equals("liquid", StringComparison.CurrentCultureIgnoreCase) ? "text/plain" : "application/xml";
```

Comment:
Is the actual ContentType parameter used anymore? I don't see any usage.

Reply:
It isn't. I debated whether accepting the customer's input would override what I can infer. I know that if the customer provides a liquid file it is 1 content type, otherwise it is the other. Should I use the customer input instead and override if only is isn't provided?

Reply:
It should be fine to no longer use the user's input.  What you should do is deprecate the ContentType parameter because you are no longer using it (https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#a-parameter-is-being-deprecated).

Reply:
I've marked both New and Set to be deprecated so that they match (instead of just Set)